### DISCUSSION TITLE
It would be cool if...

### DISCUSSION THREAD

**Discussion by coff33ninja** [YerongAI/Office-Tool#710] [Ideas]
It would be cool if we can launch this program from a command line utilizing the generated configuration saved from an earlier usage of the GUI. Given that batch deployment would be benefactor to the idea. Most of us also utilize scripts to get things done faster me included =)

**Comment by YerongAI** [comment]
If you are asking for command line, current we have provided limited supports. See docs [here](https://help.coolhub.top/others/commands.html).

For multiple deployment, I think that use [ODT](https://docs.microsoft.com/deployoffice/overview-office-deployment-tool) is better.

**Comment by alesankov** [comment]
Can anyone tell, how deploy office iso, generated in OPT without GUI?
With GUI everything works fine. I tried to use setup.exe from Files folder + xml, generated in OPT, but procces starts in CMD and ends with nothing. Also tried to use XML, generated with OCT (microsoft Generator) + setup.exe from Files folder (copied it to near XML) + setup files from OPT, but same result.
